src/Filter/Filter.php

```diff
@@ -18,7 +18,7 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-abstract class Filter implements FilterInterface
+abstract class Filter implements FilterInterface, ChainableFilterInterface
```

Comment:
If ChainableFilterInterface extends FilterInterface you don't need the both interface.

Or maybe is it better for ChainableFilterInterface to not extends FilterInterface.